prebuilt-VMs/compute-node/Vagrantfile

```diff
@@ -0,0 +1,52 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+require 'fileutils'
+
+# We need the ansible provisioner with galaxy support
+Vagrant.require_version ">= 1.8.1"
+Vagrant.configure(2) do |config|
+
+  # Customizable properties
+  vm_ip = ENV["COMPUTE_NODE_IP"] || "192.168.100.100"
+
+  # Define image
+  config.vm.box = "VirtualBox-OpenFOAM-PyFr-ParaView-v1"
+  config.vm.box_url = "https://data.kitware.com/api/v1/file/587826838d777f05f44b1423/download"
+  config.vm.network "private_network", ip: vm_ip
+
+  # Provider-specific configuration so you can fine-tune various
+  # backing providers for Vagrant. These expose provider-specific options.
+  # Example for VirtualBox:
+  # Dynamically allocate memory and cpus,
+  # see https://stefanwrobel.com/how-to-make-vagrant-performance-not-suck
+  config.vm.provider "virtualbox" do |v|
+    host = RbConfig::CONFIG['host_os']
+
+    # Give VM 1/4 system memory & access to 1/2 of the cpu cores on the host
+    if host =~ /darwin/
+      cpus = `sysctl -n hw.ncpu`.to_i
+      cpus = cpus / 2
```

Comment:
As we are telling user to run multiple VMs now, i.e. a compute and hpccloud VM, we might want to reduce the number of cpus we are giving to each VM? This doesn't have to done in this PR, but is something we should think about.